jekyll/_cci2/server-3-operator-backup-and-restore.adoc

```diff
@@ -17,20 +17,21 @@ toc::[]
 == Overview
 _Backup and restore is available for server v3.1.0 and up._
 
-While operating and administering CircleCI server, you will consider how to maintain backups and recover your installation, should there be a need to migrate it to another cluster or recover from a critical event.
+While operating and administering CircleCI server, you will need to consider how to maintain backups and recover your installation, should there be a need to migrate it to another cluster or recover from a critical event.
+
 This document outlines recommendations for how to back up and restore your CircleCI server instance data and state.
 
-CircleCI server is administered via https://kots.io/[KOTS], which uses https://velero.io/[Velero] for backup and restore.
-The benefit of this approach is that it not only restores your application's data,
+CircleCI server is administered via https://kots.io/[KOTS], which uses https://velero.io/[Velero] for backup and restore. The benefit of this approach is that it not only restores your application's data,
 but it also restores the state of the Kubernetes cluster and its resources at the time of the backup.
-In this way, we can also restore admin-console configurations and customizations you made to your cluster. 
+This means you can also restore Admin Console configurations and customizations you made to your cluster. 
 
 NOTE: Backup and restore of the CircleCI services is dependent on Velero. If your cluster is lost, you will not be able to restore CircleCI until you have successfully started Velero in the cluster. From there you can recover the CircleCI services.
 
-== The setup
+== The Setup
```

Comment:
should be sentence case - but I see all the headings are the wrong case in this doc!

Reply:
Haha, I've made them all sentence case.